drv/Maps.drv

```diff
@@ -91,23 +98,12 @@ public class MAPS {
 #endif
 
 		@Override
-		public void putAll(final Map<? extends KEY_GENERIC_CLASS, ? extends VALUE_GENERIC_CLASS> m) { throw new UnsupportedOperationException(); }
+		public void putAll(Map<? extends KEY_GENERIC_CLASS, ? extends VALUE_GENERIC_CLASS> m) { throw new UnsupportedOperationException(); }
```

Comment:
Oh well I guess there might be a billion different opinions. Do not force yourself, it's OK, but please do not delete my finals :).